pkg/installation/util.go

```diff
@@ -79,27 +80,22 @@ func getDownloadTarget(index index.Plugin) (version, uri string, fos []index.Fil
 	return version, uri, p.Files, p.Bin, nil
 }
 
-// ListInstalledPlugins returns a list of all name:version for all plugins.
-func ListInstalledPlugins(installDir, binDir string) (map[string]string, error) {
-	installed := make(map[string]string)
-	plugins, err := ioutil.ReadDir(installDir)
+// ListInstalledPlugins returns a list of all install plugins in a
+// name:version format based on the install receipts at the specified dir.
+func ListInstalledPlugins(receiptsDir string) (map[string]string, error) {
```

Comment:
Nice, that just got a lot simpler :+1: 
Would be a good time to also add some tests for this.

Reply:
Since adding tests to this requires me to Store() some manifests by serializing the in-memory Plugin objects (#270) to a tempdir, I will need #270 to happen first.

To unblock here, I'm adding a TODO to the code explaining this situation.